pkg/build/logger.go

```diff
@@ -0,0 +1,12 @@
+package build
+
+type Logger interface {
```

Comment:
There's already some logging abstraction in apko/melange, can we reuse it?

Reply:
I couldn't find an existing interface like this, but I definitely want to use what already exists where possible